crates/subspace-farmer/src/single_plot_farm.rs

```diff
@@ -332,6 +332,7 @@ pub(crate) struct SinglePlotFarmOptions<'a, RC, PF> {
 /// disk plot.
 #[must_use = "Farm does not function properly unless run() method is called"]
 pub struct SinglePlotFarm {
+    directories: Vec<PathBuf>,
```

Comment:
Since this is only stored in order to be used in benches, can we not add this and instead store it on the benching side?